concepticondata/conceptlists/Josserand-1983-188.tsv

```diff
@@ -0,0 +1,189 @@
+ID	NUMBER	SPANISH	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Josserand-1983-188-1	1	lluvia	rain	658	RAIN (PRECIPITATION)
+Josserand-1983-188-2	2	maguey	agave		
+Josserand-1983-188-3	3	petate	straw mat	195	MAT
+Josserand-1983-188-4	4	cuatro	four	1500	FOUR
+Josserand-1983-188-5	5	pluma	feather	1201	FEATHER
+Josserand-1983-188-6	6	cola	tail	1220	TAIL
+Josserand-1983-188-7	7	cera	wax	1931	BEESWAX
+Josserand-1983-188-8	8	nariz	nose	1221	NOSE
+Josserand-1983-188-9	9	palabra	word	1599	WORD
+Josserand-1983-188-10	10	está cerca	near	1942	NEAR
+Josserand-1983-188-11	11	árbol	tree	906	TREE
+Josserand-1983-188-12	12	venado	deer	1936	DEER
+Josserand-1983-188-13	13	manteca	lard	1282	FAT (FROM ANIMALS)
+Josserand-1983-188-14	14	olor	smell	2124	SMELL
+Josserand-1983-188-15	15	tortilla	tortilla		
+Josserand-1983-188-16	16	vomita	vomits	1278	VOMIT
+Josserand-1983-188-17	17	flojo	lazy	1564	LAZY
+Josserand-1983-188-18	18	cama	bed	1663	BED
+Josserand-1983-188-19	19	comal	griddle	31	PAN
+Josserand-1983-188-20	20	culpa	sin	2971	SIN
+Josserand-1983-188-21	21	frijol	bean	832	BEAN
+Josserand-1983-188-22	22	agua	water	948	WATER
+Josserand-1983-188-23	23	río	river	3311	FLOWING BODY OF WATER
+Josserand-1983-188-24	24	lodo	mud	640	MUD
+Josserand-1983-188-25	25	hermano de hombre	brother of man	559	BROTHER (OF MAN)
+Josserand-1983-188-26	26	tres	three	492	THREE
+Josserand-1983-188-27	27	diente	tooth	1380	TOOTH
+Josserand-1983-188-28	28	cinco	five	493	FIVE
+Josserand-1983-188-29	29	amargo	bitter	887	BITTER
+Josserand-1983-188-30	30	carne	meat	2615	FLESH OR MEAT
+Josserand-1983-188-31	31	rojo	red	156	RED
+Josserand-1983-188-32	32	se fue	(he) went	695	GO
+Josserand-1983-188-33	33	siete	seven	1704	SEVEN
+Josserand-1983-188-34	34	hilo	thread	1161	THREAD
+Josserand-1983-188-35	35	animal	animal	619	ANIMAL
+Josserand-1983-188-36	36	se acaba	it finishes	1766	FINISH
+Josserand-1983-188-37	37	blanquillo	egg	744	EGG
+Josserand-1983-188-38	38	espuma	foam	1768	FOAM
+Josserand-1983-188-39	39	sangre	blood	946	BLOOD
+Josserand-1983-188-40	40	marido	husband	1200	HUSBAND
+Josserand-1983-188-41	41	sal	salt	1274	SALT
+Josserand-1983-188-42	42	red	net bag	293	NETBAG
+Josserand-1983-188-43	43	jícara	gourd	411	GOURD
+Josserand-1983-188-44	44	calabaza	squash		
```

Comment:
The reason I didn't map it to pumpkin is that the orange, round veggie typically associated with pumpkin doesn't exist in the region (or at least is not the default). The squashes are all quite different. But of course you know best how to decide in such cases.

Reply:
The definition of the concept gloss is: "A squash fruit of the Cucurbita genus, most commonly orange in colour when ripe and traditionally used during Halloween." 
I think it would be ok to map this form to PUMPKIN. Although the definition describes the most "prototypical" kind, other mappings such as the German "Kürbis" can also refer to all kinds of squash/pumpkin.